docs/paper/dev/guides/pdc.md

```diff
@@ -0,0 +1,148 @@
+---
+slug: /dev/pdc
+---
+
+# Persistent Data Container (PDC)
+
+The Persistent Data Container (PDC) is a way to store custom data on a whole range of objects; such as items, entities, and blocks.
+The Full list of classes that support the PDC are:
+
+- [Chunk](#chunk)
+- [World](#world)
+- [Entity](#entity)
+- [TileState](#tilestate)
+- [ItemMeta](#itemmeta)
+- [Structure](#structure)
+
+## What is it used for?
+In the past, developers resorted to storing data within the NBT tags of an object. 
```

Comment:
It might be nice to also include stuff like "checking the lore" or "checking the display name" here.
E.g. parsing integers from the lore.

Plugin made NBT tags were already the "better" solution lol :joy: most people we direct to the PDC come from a lore or display name usage.

Reply:
It's sunday and not a wordsmith, but, this entire section is far from ideal, IMHO;
the nature of the PDC being better than NBT is really just about removing the need for abstractions/unstable reflection; it being "hacky" is really just weird, the only real place that supported arbitrary data in general was itemstacks, and that was purely on the foundation that the item tag stored arbitrary data anyways; until you get to the other things, entities, chunks, etc, which had 0 way to store arbitrary data on them in general